packages/file-asset-apis/src/file-api/file-sender-api.ts

```diff
@@ -0,0 +1,60 @@
+import {
+    RouteSenderAPI,
+    DataEntity,
+    Logger,
+    TSError
+} from '@terascope/job-components';
+import fse from 'fs-extra';
+import { ChunkedFileSender } from '../lib';
+import { ChunkedSenderConfig, FileSenderType } from '../interfaces';
+
+export class FileSender extends ChunkedFileSender implements RouteSenderAPI {
+    logger: Logger;
+    concurrency: number;
+
+    constructor(config: ChunkedSenderConfig, logger: Logger) {
+        super(FileSenderType.file, config);
+        this.logger = logger;
+        const { concurrency } = config;
+        this.concurrency = concurrency;
+    }
+    /**
+     * This is a low level API, it is not meant to be used externally,
+     * please use the "send" method instead
+     * @protected
+     * @param {string} path
+     * @param {((DataEntity | Record<string, unknown>)[])} records
+     * @returns {Promise<any>}
```

Comment:
It is typescript you don't need to define the `@returns` `@protected` `@param` or `@memberof` since typescript manages that. Take a look at https://tsdoc.org/ for more information.